### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by tdincer**

### CODE REVIEW

File: element_calcium_imaging/scan.py [hunk 1]
```diff
@@ -205,7 +205,7 @@ class ScanLocation(dj.Manual):
 
 @schema
 class ScanInfo(dj.Imported):
-    """Metadata of the scan file
+    """Metadata of scans
 
     Refer to the `definition` attribute for the table design."""
```
Comment (dimitri-yatsenko) [inline element_calcium_imaging/scan.py:210]:
```suggestion
    """
    Information about the scan extracted from the recorded files. 
    Refer to the `definition` property to see the table design.
    """
```

File: docs/src/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Element Calcium Imaging
+
+DataJoint element for functional calcium imaging with `ScanImage`, `Scanbox`,
+`Nikon NIS`, or `PrairieView` acquisition systems and `Suite2p` or `CaImAn` suites
+for analysis. DataJoint Elements collectively standardize and automate data collection
+and analysis for neuroscience experiments. Each Element is a modular pipeline for data
+storage and processing with corresponding database tables that can be combined with
+other Elements to assemble a fully functional pipeline.
```
Comment (kabilar) [inline docs/src/index.md:8]:
```suggestion
DataJoint Element for functional calcium imaging with `ScanImage`, `Scanbox`,
`Nikon NIS`, or `PrairieView` acquisition software and `Suite2p` or `CaImAn` analysis software. DataJoint Elements collectively standardize and automate data collection
and analysis for neuroscience experiments. Each Element is a modular pipeline for data
storage and processing with corresponding database tables that can be combined with
other Elements to assemble a fully functional pipeline.
```

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @tdincer!  Minor comments below.  I will come back to review the `Concepts` page.

File: docs/src/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# Installation
```
[the remark this answers is not in the document]
Comment (tdincer) [reply docs/src/getting_started.md]: This file doesn't exist anymore.

File: docs/src/tutorials.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# Tutorials
```
[the remark this answers is not in the document]
Comment (tdincer) [reply docs/src/tutorials.md:1]: Emptied the content of tutorials.

File: docs/src/getting_started.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kabilar) [reply docs/src/getting_started.md]: Thank you

File: docs/src/tutorials.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (kabilar) [reply docs/src/tutorials.md:1]: Thank you

File: docs/src/citation.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Citation
+
+If your work uses this Element, please cite the following manuscript and Research
+Resource Identifier (RRID).
+
++ Yatsenko D, Nguyen T, Shen S, Gunalan K, Turner CA, Guzman R, Sasaki M, Sitonic D,
+  Reimer J, Walker EY, Tolias AS. DataJoint Elements: Data Workflows for
+  Neurophysiology. bioRxiv. 2021 Jan 1. doi: https://doi.org/10.1101/2021.03.30.437358
+
++ DataJoint Elements ([RRID:SCR_021894](https://scicrunch.org/resolver/SCR_021894)) -
```
Comment (CBroz1) [inline docs/src/citation.md:10]: Please add [automated versioning to the citation page](https://github.com/datajoint/element-deeplabcut/commit/352c44f98325cb79a249026d84310bff8417e7e2)

Comment (tdincer) [reply docs/src/citation.md:10]: Added it at https://github.com/datajoint/element-calcium-imaging/pull/92/commits/577173d39bd89127333d5fafc719aa413ab07044

**Pull request closed without merging**